src/components/chat/MessageForm.js

```diff
@@ -132,14 +133,23 @@ class MessageForm extends Component {
     this.messageInput = input;
   }
 
+  onMessageUpdate = (e) => {
+    const message = e.target.value;
+    this.props.messageUpdate(message);
+  }
```

Comment:
Even if we don't ship this feature, I'd like to update this at some point. IMHO, the redux actions should not be coupled with browser event handlers directly.